src/entity/Living.php

```diff
@@ -184,6 +184,13 @@ protected function addAttributes() : void{
 		$this->attributeMap->add($this->absorptionAttr = AttributeFactory::getInstance()->mustGet(Attribute::ABSORPTION));
 	}
 
+	/**
+	 * Returns the display name of this entity to use in stuff like death messages.
```

Comment:
```suggestion
	 * Returns the name used to describe this entity in chat and command outputs.
```
In the future we'll most likely use this for things like `/kill` and the like.

Reply:
done